stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,306 @@ td.table__form .rules + .form__group:last-of-type {
         width: auto;
     }
 }
+
+/**
+ * The following version 2 was developed to meet the needs of XFP conditional logic rules
+ */
+.rules-container--v2 {
```

Comment:
Small thing but maybe something a little less versioned? `rule-container--logic` perhaps?

Reply:
already being implemented in-product so will leave as-is for now